app/assets/javascripts/components/sampleUpload.jsx

```diff
@@ -217,7 +217,7 @@ class SampleUpload extends React.Component {
 
   filePathValid(str) {
     var regexPrefix = /^s3:\/\//;
```

Comment:
i think you can allow the white space in the beginning because the backend automatically trims it.